internal/workflow/preprocessing.go

```diff
@@ -1,25 +1,44 @@
 package workflow
 
 import (
-	"errors"
 	"fmt"
 	"path/filepath"
+	"strings"
 	"time"
 
 	"github.com/artefactual-sdps/temporal-activities/bagit"
 	"go.artefactual.dev/tools/temporal"
+	temporalsdk_log "go.temporal.io/sdk/log"
 	temporalsdk_temporal "go.temporal.io/sdk/temporal"
 	temporalsdk_workflow "go.temporal.io/sdk/workflow"
 
 	"github.com/artefactual-sdps/preprocessing-sfa/internal/activities"
+	"github.com/artefactual-sdps/preprocessing-sfa/internal/enums"
+	"github.com/artefactual-sdps/preprocessing-sfa/internal/eventlog"
+)
+
+type Outcome int
+
+const (
+	OutcomeSuccess Outcome = iota
+	OutcomeSystemError
+	OutcomeContentError
 )
```

Comment:
I like the outcome being an `int`, it should be easier to document and match in enduro.